locale/pl/LC_MESSAGES/client.po

```diff
@@ -95,7 +95,7 @@ msgstr "Przykro nam o tym słyszeć. Jeśli nie ufasz już temu urządzeniu, to
 
 #: .es5/scripts/views/settings/clients.js:52
 msgid "Last active %(translatedTimeAgo)s"
-msgstr ""
+msgstr "Ostatnia aktywność: %(translatedTimeAgo)s"
```

Comment:
The `%(translatedTimeAgo)s` gets replaced with a translated string along the lines of "5 days ago", which is generated by a third-party library:

  http://momentjs.com/docs/#/displaying/fromnow/